README.md

```diff
@@ -1,16 +1,80 @@
+    - 10 minutes for discussing the submitted solution. Get the candidate to explain design decisions for the code and the tests
+    - 5 minutes to ask some more critical, challenging questions about the solution
+    - 30 minutes for adapting the solution to a change in requirements. Make up a change that will necessitate the code to be adapted, and ask the candidate to implement changes to make the solution conform to the new requirements. Offer guidance if needed, but if they're really struggling and taking the full time up then don't go beyond the 30 minutes.
+    - 10 minutes to answer a couple of general software engineering theory questions
+    - 5 minutes for the candidate to ask questions
+3. Collate feedback offline and return to the candidate. This could be either via a follow up call or another format if convenient.
+
+### Option 2: Live Coding Interview
+
+This requires more time than the above option and is more intense, but has the advantage of being a more interactive approach and prepares candidates well for more collaborative interviews.
+
+1. Select a Live Assignment and explain it to the candidate. Observe as he/she implements a solution, offering clarity about the exercise and its requirements, but please don't offer programming assistance.
+2. Present a change in requirements to the candidate. Assess how he/she responds to tweak the design and implementation of the solution. Ask the interviewee to explain the choices made in adapting the solution to fit the new problem scope.
+3. Allow roughly a 10 to 15 minute time slot at the end of the interview for questions. Set approximately:
+    - 10 minutes for the candidate to answer a couple of general software engineering theory questions
+    - 5 minutes for the candidate to ask questions
+4. Feedback can be given at the end, immediately after the interview, or collated offline and fed back in whatever format is deemed most appropriate.
+
+## How to Assess and Provide Feedback
+
+As outlined previously, the CYF mock interview process is designed to assess a candidate's all-round ability and feed this back in a manner that gives candidates **clarity about what needs to improve and why** as well as **confidence to keep doing the things they're doing well**.
+
+Here are some general pointers that interviewers should bear in mind during the mock interview process:
+
+* Take notes throughout the interview - this enables you to provide good feedback about specific, relatable things.
+* Ensure to highlight what went well rather than just what needs to be improved upon.
+* Be open and honest - this is a learning experience for the candidates, so there's unlikely to be any benefit from holding anything back.
+
+The format of the feedback can be whatever is most convenient, but generally conversational feedback has less chance of being misunderstood than written feedback, which can sometimes be taken in the wrong way.
+
```

Comment:
Missed this one

Reply:
What's this one, sorry?

Reply:
Please remove the empty line at the end of the file